modules/python/package/setup.py

```diff
@@ -1,9 +1,19 @@
 import os
-import sys
-import platform
 import setuptools
 
-SCRIPT_DIR=os.path.dirname(os.path.abspath(__file__))
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
To succeed with merge we need to be ready for features on 5.x branch.
The source of problem is here: #19156

Possible way 1:
- backport from 5.x this feature (only python part) and add some standalone test for python bindings
- update PR to handle "named arguments"

Possible way 2:
- open a second PR for 5.x branch with typing stubs update which handles "named arguments"

Cons 2:
- synchronization issues between 4.x and 5.x PRs
- no Python PRs during the "Merge 4.x" as changes from 4.x are ignored (changes from .x PR are used).

Cons 1:
- extra work to add "standalone" Python binding test (without modification of OpenCV C++ API).

Reply:
Named arguments are unrolled as expected after PR #19156 is merged

```python
# Functions
@typing.overload
def copyMatAndDumpNamedArguments(src: cv2.typing.MatLike, dst: cv2.typing.MatLike | None = ..., lambda_: int = ..., sigma: float = ...) -> tuple[str, cv2.typing.MatLike]: ...
@typing.overload
def copyMatAndDumpNamedArguments(src: cv2.UMat, dst: cv2.UMat | None = ..., lambda_: int = ..., sigma: float = ...) -> tuple[str, cv2.UMat]: ...
```